src/index.ts

```diff
@@ -1,4 +1,13 @@
-console.log("[ITEM-STORE] Starting item store...");
+import { logger, logModel } from "./services/logger";
+
+logModel.service = "[ITEM-STORE]"
+logModel.date = new Date();
+logger.log({
+    level: 'info',
+    logInfos: logModel,
+    message: "Starting item store...",
+    correlationId: '123',
```

Comment:
let's just remove correlationId for now, we'll add it in the relevant piece of work when it's generated automatically. It's better than having a static one